packages/atomic/src/components/insight/user-actions/atomic-insight-user-action/atomic-insight-user-action.tsx

```diff
@@ -0,0 +1,97 @@
+import {FunctionalComponent, h} from '@stencil/core';
+import {UserAction} from '../..';
```

Comment:
Huh? This will come from Headless when Simon's PR is merged right?

Reply:
exactly x)